web/src/components/welcome-modal/welcome-modal.tsx

```diff
@@ -22,8 +25,28 @@ export interface WelcomeModalProps extends ModalProps {
 export default ({ challengeToken, teamToken, ...props }: WelcomeModalProps) => {
   const readableTeamName = challengeTeams[teamToken].readableName;
   const [hasAgreed, setHasAgreed] = useState<boolean>(false);
+  const account = useAccount();
+  const saveAccount = useAction(User.actions.saveAccount);
+
   useEffect(() => trackChallenge('modal-welcome'), []);
 
+  const parseEnrollment = (queryString: string): Enrollment => {
+    const regex = new RegExp(/([a-z]+)=([a-z]+)/, 'gm');
```

Comment:
I don't think `[a-z]+` will match our invite codes, which look like: `e3cdefb8-0af0-11ea-b89e-06dc45d2f642`.

There's a chance that invite codes (or team names) will contain capitals or other characters in the future, so perhaps `[^=]` is more durable?

Also re: `'gm'`, does this need to be multiline?